src/AuditReader.php

```diff
@@ -216,6 +216,8 @@ public function find($className, $id, $revision, array $options = [])
                 $columnName = $idKeys[0];
             } elseif (isset($classMetadata->fieldMappings[$idField])) {
                 $columnName = self::getMappingColumnNameValue($classMetadata->fieldMappings[$idField]);
+            } elseif ($this->config->areAssociationsDisabled()) {
+                continue;
             } elseif (isset($classMetadata->associationMappings[$idField]['joinColumns'])) {
```

Comment:
I prefer
```
} elseif (isset($classMetadata->associationMappings[$idField]['joinColumns'])) {
    if ($this->config->areAssociationsDisabled()) {
                continue;
    }
    $columnName = $classMetadata->associationMappings[$idField]['joinColumns'][0]['name'];
}
```

Or you will silence error for column not found when you disable associations